src/backend/api/Fusion.Resources.Api/Controllers/Personnel/PersonnelController.cs

```diff
@@ -0,0 +1,148 @@
+            var personnel = new Faker<ApiContractPersonnel>()
+                .RuleFor(p => p.AzureUniquePersonId, f => f.PickRandom<Guid?>(new[] { (Guid?)null, Guid.NewGuid(), Guid.NewGuid(), Guid.NewGuid(), Guid.NewGuid() }))
+                .RuleFor(p => p.Name, f => f.Person.FullName)
+                .RuleFor(p => p.Mail, f => f.Person.Email)
+                .RuleFor(p => p.JobTitle, f => f.Name.JobTitle())
+                .RuleFor(p => p.PhoneNumber, f => f.Person.Phone)
+                .RuleFor(p => p.HasCV, f => f.Random.Bool())
+                .RuleFor(p => p.AzureAdStatus, f => f.PickRandomWithout<ApiContractPersonnel.ApiAccountStatus>(ApiContractPersonnel.ApiAccountStatus.NoAccount))
+                .FinishWith((f, p) =>
+                {
+                    if (p.AzureUniquePersonId == null)
+                    {
+                        p.AzureAdStatus = ApiContractPersonnel.ApiAccountStatus.NoAccount;
+                    }
+
+                    p.Disciplines = Enumerable.Range(0, f.Random.Number(1, 4)).Select(i => new PersonnelDiscipline { Name = f.Hacker.Adjective() }).ToList();
+                })
+                .Generate(new Random().Next(50, 200));
+
+            var collection = new ApiCollection<ApiContractPersonnel>(personnel);
+            return collection;
+        }
+
+        [HttpPost("/projects/{projectIdentifier}/contracts/{contractIdentifier}/resources/personnel")]
+        public async Task<ActionResult<ApiContractPersonnel>> CreateContractPersonnel(string projectIdentifier, string contractIdentifier, [FromBody] CreateContractPersonnelRequest request)
+        {
+
+            var person = new Faker<ApiContractPersonnel>()
+                .RuleFor(p => p.AzureUniquePersonId, f => f.PickRandom<Guid?>(new[] { (Guid?)null, Guid.NewGuid() }))
+                .RuleFor(p => p.AzureAdStatus, f => f.PickRandom<ApiContractPersonnel.ApiAccountStatus>())
+                .FinishWith((f, p) => p.AzureAdStatus = p.AzureUniquePersonId == null ? ApiContractPersonnel.ApiAccountStatus.NoAccount : p.AzureAdStatus)
+                .Generate();
+
+
+            var item = new ApiContractPersonnel()
+            {
+                AzureAdStatus = person.AzureAdStatus,
+                AzureUniquePersonId = person.AzureUniquePersonId,
+                Name = request.Name,
+                JobTitle = request.JobTitle,
+                Mail = request.Mail,
+                PhoneNumber = request.PhoneNumber
+            };
+            
+            return Created($"/projects/{projectIdentifier}/contracts/{contractIdentifier}/resources/personnel/{item.Mail}", item);
+        }
+
+        [HttpPost("/projects/{projectIdentifier}/contracts/{contractIdentifier}/resources/personnel-collection")]
+        public async Task<ActionResult<ApiBatchResponse<ApiContractPersonnel>>> CreateContractPersonnelBatch(string projectIdentifier, string contractIdentifier, [FromBody] IEnumerable<CreateContractPersonnelRequest> requests)
```

Comment:
Does dotnet mvc handle `[FromBody] IEnumerable<T>`? I think that I've had to wrap enumerables in another type before 🤔

Reply:
didn't test, will find out :) anyway just initial setup.